staging/src/k8s.io/client-go/tools/leaderelection/resourcelock/leaselock.go

```diff
@@ -101,6 +101,11 @@ func (ll *LeaseLock) Identity() string {
 	return ll.LockConfig.Identity
 }
 
+// Key returns the Key of the lock
+func (ll *LeaseLock) Key() string {
```

Comment:
Calling `Update()` or `Create()` on the LeaseLock leaves out the `HolderKey`. Lease lock is different from the rest of the resource lock impls since it uses the lease spec instead of cramming all the fields in an annotation so we need to find a new place to put the key (putting `HolderKey` and priority as a concept in `v1.Lease` seems wrong but I'm not super clear on what lease gets used for in k8s besides kubelet heartbeats).